src/configuration/services/postgresql.md

```diff
@@ -259,62 +256,11 @@ If you see this error: ``Fatal error: Uncaught exception 'PDOException' with mes
 
 ## Upgrading
 
-Although PostgreSQL includes the `pg_upgrade` command for direct migration of data between versions, the process for using it requires both the old and new versions to be installed side by side, and sometimes requires user intervention. For that reason, we don't support transparent updates from one PostgreSQL version to another (such as from 9.3 to 9.6). Accordingly, upgrading between major versions of PostgreSQL requires a manual dump and restore of the data.
-
-To minimize the downtime associated with such upgrades, we suggest the following process:
-
-1. Create a development environment that you can use to test the upgrade process.
-2. Create a new service for the new PostgreSQL version (alongside the existing one), and point your application container's existing relationship at the new service.
-3. Add a new relationship to expose the service running the older PostgreSQL version
-4. Add a [deploy hook](#example-deploy-hook) that migrates the data from the old service to the new one. This prevents the application from loading and modifying the database during the migration.
-5. Make a Git commit with these changes, and push it into the development environment that you're testing with.
-6. In the development environment, review `/var/log/deploy.log` to confirm that the migration was successful, inspect the database, and test your application as needed to confirm that the migration was successful and that the application works with the new PostgreSQL version.
-7. When you're ready to upgrade the database on your master environment, start off by triggering a fresh snapshot of the master environment's data. This will give you the ability to roll back the upgrade process in case something goes wrong.
-8. Push the same commit from step 5 into your project's master branch. This will carry out the migration in your production environment.
-9. Finally, push out another commit that removes the old database service and the deploy hook used for migration, which is no longer needed now that the upgrade process is finished.
+Simply change the version in your services.yaml file, Platform.sh will transparently migrate the data. 
```

Comment:
It only works for pg10. Upgrading from, say, 9.3 to 9.6 still doesn't work.

Reply:
but both 9.3 to 10 and 9.6 to 10.0 would work right ? if so let's just say upgrading from 9.3 to 9.6 is unsupported and be done with it.

Reply:
I'm fine with just declaring 9.3->9.6 unsupported, but this comment isn't resolved until the text is updated to reflect that.